### PR TITLE
#2194 add ClipAction.getAnimClip() method, remove ClipAction.reset() method

### PR DESCRIPTION
**Pull request by capdevon**
add jMonkeyEngine license

### CODE REVIEW

File: jme3-core/src/main/java/com/jme3/anim/tween/action/ClipAction.java [hunk 1]
```diff
@@ -59,8 +96,8 @@ private void interpolateMorphTrack(double t, MorphTrack track) {
 //        }
     }
 
-    public void reset() {
-
+    public AnimClip getAnimClip() {
```
Comment (Scrappers-glitch) [inline jme3-core/src/main/java/com/jme3/anim/tween/action/ClipAction.java:99]: Could you specify a JavaDoc here, please? 
For example: 
```java
/** 
 * Retrieves the animation clip (containing the animation tracks) 
 * associated with this action.
 * 
 * @return the associated animation clip object
 */
``` 
Thanks.

**Review by Scrappers-glitch — APPROVED** [review]

**Comment by scenemax3d** [comment]
Hi @capdevon ,
Can you explain why did you remove the reset() method?

Thanks!

**Comment by capdevon** [comment]
Hi @scenemax3d , the `reset `method is **empty** and does **nothing**. 
It is a method that was left incomplete in the transition to the new animation system. 
Removing it does not cause any problem. In my opinion it only creates confusion and lowers the quality of the code.

**Comment by Scrappers-glitch** [comment]
Yes, either to implement the method or remove it completely, however implementation will require a discussion on forum and some prerequisites, and thus another completely different PR. @scenemax3d If you want to support resetting ClipActions, then you might consider opening an issue for a later implementation.

**Pull request merged**